server/src/main/java/com/cloud/network/router/VirtualNetworkApplianceManager.java

```diff
@@ -36,82 +36,83 @@
  */
 public interface VirtualNetworkApplianceManager extends Manager, VirtualNetworkApplianceService {
 
-    static final String RouterTemplateXenCK = "router.template.xenserver";
-    static final String RouterTemplateKvmCK = "router.template.kvm";
-    static final String RouterTemplateVmwareCK = "router.template.vmware";
-    static final String RouterTemplateHyperVCK = "router.template.hyperv";
-    static final String RouterTemplateLxcCK = "router.template.lxc";
-    static final String RouterTemplateOvm3CK = "router.template.ovm3";
-    static final String SetServiceMonitorCK = "network.router.EnableServiceMonitoring";
-    static final String RouterAlertsCheckIntervalCK = "router.alerts.check.interval";
-    static final String VirtualRouterServiceOfferingCK = "router.service.offering";
-
-    static final String RouterHealthChecksConfigRefreshIntervalCK = "router.health.checks.config.refresh.interval";
-    static final String RouterHealthChecksResultFetchIntervalCK = "router.health.checks.results.fetch.interval";
-    static final String RouterHealthChecksFailuresToRecreateVrCK = "router.health.checks.failures.to.recreate.vr";
-
-    static final ConfigKey<String> RouterTemplateXen = new ConfigKey<String>(String.class, RouterTemplateXenCK, "Advanced", "SystemVM Template (XenServer)",
+    String RouterTemplateXenCK = "router.template.xenserver";
+    String RouterTemplateKvmCK = "router.template.kvm";
+    String RouterTemplateVmwareCK = "router.template.vmware";
+    String RouterTemplateHyperVCK = "router.template.hyperv";
+    String RouterTemplateLxcCK = "router.template.lxc";
+    String RouterTemplateOvm3CK = "router.template.ovm3";
+    String SetServiceMonitorCK = "network.router.EnableServiceMonitoring";
+    String RouterAlertsCheckIntervalCK = "router.alerts.check.interval";
+    String VirtualRouterServiceOfferingCK = "router.service.offering";
+
+    String RouterHealthChecksConfigRefreshIntervalCK = "router.health.checks.config.refresh.interval";
+    String RouterHealthChecksResultFetchIntervalCK = "router.health.checks.results.fetch.interval";
+    String RouterHealthChecksFailuresToRecreateVrCK = "router.health.checks.failures.to.recreate.vr";
+    String RemoveNicsOnStopCK = "ssvm.release.control.ip.on.stop";
```

Comment:
SystemVmReleaseControlIpOnStop

looks more accurate

Reply:
Since it is applicable for all system vms and virtual routers,  does it make sense move the definition to another class?

Reply:
```suggestion
    String RemoveNicsOnStopCK = "systemvm.release.control.ip.on.stop";
```